src/main/java/org/bookstore/repository/BookRepository.java

```diff
@@ -0,0 +1,11 @@
+package org.bookstore.repository;
+
+import java.util.List;
+import org.bookstore.model.Book;
+
+public interface BookRepository {
+
```

Comment:
To be honest  I don’t prefer to leave one empty line after class declaration but you can do it if you like it BUT!!! make it consistent (leave one empty line in each class or remove it everywhere)